index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="styles/style.css" rel="stylesheet">
+</head>
+
+<body class="index-grid">
+  <div>
+    <nav class="header">
+      <ul>
+        <li><a href="index.html" class="text-light">Ruge Li</a></li>
+        <li><a href="portfolio.html" class="text-light">Portfolio</a></li>
+        <li><a href="about.html" class="text-light">About Me</a></li>
+      </ul>
+    </nav>
+  </div>
+
+  <div class="homepage">
+    <p><span>while (alive){</span>
+          <span>sleep();</span>
+          <span>eat();</span>
+          <span>code();</span>
+          <span>petMyDog();}</span>
+      </p>
+    </div>
+
+  <section>
+    <div class="contact-me">
+      <h1 class="my-name">I AM RUGE LI.</h1>
+      <button class="btn">
+        Contact me
+      </button>
+    </div>
+  </section>
+
+
+  <section class="footer">
+		<div class="column-left">
+		</div>
+		<div class="column-main">
+			<h3 class="footer-text">&#169; Ruge Li. All Rights Reserved. 2022. </h3>
```

Comment:
In semantic HTML, header tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)